dev-docs/bidders/nexx360.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: bidder
+title: Nexx360
+description: Prebid Nexx360 Bidder Adapter
+pbjs: true
+biddercode: nexx360
+aliasCode: revenuemaker
```

Comment:
Please remove this line - the code is a standalone adapter, not an alias

Reply:
Done!